acceptance/lib/puppet/acceptance/common_utils.rb

```diff
@@ -1,24 +1,10 @@
-require 'puppet/acceptance/puppet_helpers'
-extend Puppet::Acceptance::PuppetHelpers
-
 module Puppet
   module Acceptance
-    module CAUtils
```

Comment:
@richardc I agree that the name-spacing is not ideal. We are working on pulling this stuff out into an external gem, but it is not of high priority at the moment.

This PR is a carbon copy of `lib/puppet/acceptance/common_utils.rb` as it exists currently in `facter`.